vendor.conf

```diff
@@ -30,7 +30,7 @@ github.com/moby/buildkit aaff9d591ef128560018433fe61beb802e149de8
 github.com/tonistiigi/fsutil dea3a0da73aee887fc02142d995be764106ac5e2
 
 #get libnetwork packages
-github.com/docker/libnetwork ab0835fffd654ce09b9e145d795c966ea2460eae https://github.com/resin-os/balena-libnetwork
+github.com/docker/libnetwork 2facb1986b84b2fcedf91dcbfdef72ab7aaf0667 https://github.com/resin-os/balena-libnetwork
```

Comment:
Updated commit hash for the balena-libnetwork commit that does the balenaEngine renaming